dev/migrate-mls

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+
+dev/run --create-mls-migration "$@"
```

Comment:
Thanks for adding this. Totally slipped my mind when I was setting up the new DB